packages/terra-list/src/Section.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ListSectionHeader from './ListSectionHeader';
+import ListItem from './ListItem';
+
+const propTypes = {
+  /**
+   * The header content to be placed for display in a section.
+   */
+  headerContent: PropTypes.string,
+  /**
+   * The list of elements to be placed for display in a section.
+   */
+  listItems: PropTypes.array,
+};
+
+const defaultProps = {
+  headerContent: '',
+  listItems: [],
+};
+
+class Section extends React.Component {
+  constructor(props) {
+    super(props);
+    this.generateHeader = this.generateHeader.bind(this);
+  }
+
+  generateHeader() {
+    return <ListItem isSelectable content={<ListSectionHeader id={this.props.headerContent} title={this.props.headerContent} onClick={this.handleClick} />} key={this.props.headerContent} />;
```

Comment:
Remove isSelectable for static headers. May need to flex this later for collapse.

Reply:
Are you suggesting to include isSelectable when implementing accordion section headers which have the collapse and expand functionality?

Reply:
We don't need to add the flexing of the prop, but I assume the isSelectable prop would be enabled with the expand/collapse functionality as it's likely the whole row would be a touch point.

Reply:
Yes, you are right, in the proof of concept I designed, I was using, isSelectableProp and used a handleClick function like this - 

```
  handleClick() {
    this.setState(prevState => ({ isOpen: !prevState.isOpen }));
  }
```

Anyways, I will go ahead and remove isSelectable prop for now.